cluster/operator-install.sh

```diff
@@ -20,6 +20,8 @@ export VERSION=v0.0.0
 export VM_IMPORT_CONFIG_STATUS=${VM_IMPORT_CONFIG_STATUS:-''}
 
 # Create and wait for the operator
+cat tests/os-mapping/operator.yaml.patch >> _out/vm-import-operator/${VERSION}/operator.yaml
```

Comment:
what is this for ?

Reply:
That's for the migration from the environment variables functional test - it adds the OS map-related  env variables to the vm-import-operator deployment (which are no longer there) so it is possible to verify that they were migrated to the config map - in https://github.com/kubevirt/vm-import-operator/blob/a7785a22a286f2ce863ee349534a9927e929c037/tests/ovirt/user_provided_os_maps_migration_test.go.